docs/source/notebooks/wps_evaluate_rules_demo.ipynb

```diff
@@ -0,0 +1,154 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluate Rules\n",
+    "Takes 2 json files - parse tree dictionary and variable dictionary - and a rule string. Returns a truth value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from birdy import WPSClient\n",
+    "from pkg_resources import resource_filename\n",
+    "from tempfile import NamedTemporaryFile\n",
+    "import os\n",
+    "import json\n",
+    "\n",
+    "# Ensure we are in the working directory with access to the data\n",
+    "if os.path.basename(os.getcwd()) != \"sandpiper\":\n",
+    "    os.chdir('../../../')\n",
+    "\n",
+    "url = 'http://localhost:5003/wps'\n",
```

Comment:
While this is going to change soon, I would like for this url to target the docker instance. That way the `make` command will work as intended. Note that because the docker target needs an updated image this will not work right away. We will have to do some updating in the containers to have the changes pulled in.